druid/examples/aspect_ratio_box.rs

```diff
@@ -0,0 +1,53 @@
+use druid::{
+    widget::{AspectRatioBox, Flex, Label, LineBreaking, MainAxisAlignment, SizedBox},
+    AppLauncher, Color, Data, Env, Lens, Widget, WidgetExt, WindowDesc,
+};
+
+fn main() {
+    let window = WindowDesc::new(ui());
+    let fixed_message =
+        "Hello there, this is a fixed size box and it will not change no matter what.";
```

Comment:
Maybe just a nitpick, but this isn't true no matter what. `SizedBox` is respecting `BoxConstraints::max`:
![image](https://user-images.githubusercontent.com/6700184/118021250-e0a93900-b30f-11eb-9a58-8eebace68412.png)